packages/vsce/__tests__/__e2e__/02_AddWiremockConfigurationProjectLevel.test.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { assert, expect } from "chai";
+import { ActivityBar, DefaultTreeSection, EditorView, InputBox, TreeItem, VSBrowser } from "vscode-extension-tester";
+
+describe("Create Project Level WireMock File Scenario", () => {
+  let cicsTree: DefaultTreeSection;
+  let quickPick: InputBox;
+  let editorView: EditorView;
+
+  before(async function () {
+    // switch to the cics extension view
+    const zoweExplorer = await new ActivityBar().getViewControl("Zowe Explorer");
+    assert(zoweExplorer !== undefined);
+    const view = await zoweExplorer.openView();
+    cicsTree = await view.getContent().getSection("cics");
+
+    // click the plus icon in cics
+    await cicsTree.click();
+    await cicsTree.expand();
+    const plusIcon = await cicsTree.getAction(`Create a CICS Profile`);
+    assert(typeof plusIcon !== "undefined");
+    await plusIcon.click();
+  });
+
+  it("Quick Pick Option test", async () => {
+    // Find quickpick
+    quickPick = await InputBox.create();
+    const qpItems = await quickPick.getQuickPicks();
+    const qpLen = qpItems.length;
+
+    // selecting wiremock_server from the quickpick
+    for(let i=0; i<qpLen; i++){
+        const wiremocklabel = await qpItems[i].getLabel();
+        if(wiremocklabel.includes("wiremock_server")){
+            expect(wiremocklabel).contains("wiremock_server")
+            await quickPick.selectQuickPick(i);
+            break;
+        }
```

Comment:
a tiny tweak that would help us diagnose things in the future - i think before we finish this for loop we should fail if we haven't found the wiremock_server entry?

that way rather than waiting to fail in the next section when we haven't got it in the list of profiles, we would find out that the problem was it wasn't in the picker of possible profiles.

does that make sense?